sdk/communication/communication-identity/src/communicationIdentityClient.ts

```diff
@@ -142,9 +151,11 @@ export class CommunicationIdentityClient {
    * @param options - Additional options for the request.
    */
   public createUser(options: OperationOptions = {}): Promise<CommunicationUserIdentifier> {
+    const operationOptions: CommunicationIdentityIssueAccessTokenOptionalParams = options;
+    operationOptions.expiresInMinutes = undefined;
     return tracingClient.withSpan(
       "CommunicationIdentity-createUser",
-      options,
+      operationOptions,
       async (updatedOptions) => {
         const result = await this.client.communicationIdentityOperations.create(updatedOptions);
```

Comment:
this method accepts a different type of options than `CommunicationIdentityIssueAccessTokenOptionalParams`, we should't mix them up

Reply:
I used another approach. Don't need to touch this now.